zellij-tile/src/shim.rs

```diff
@@ -37,6 +45,14 @@ pub fn open_file(path: &Path) {
 pub fn set_timeout(secs: f64) {
     unsafe { host_set_timeout(secs) };
 }
+pub fn exec_cmd(cmd: &[String]) -> Result<(), PermissionDenied> {
```

Comment:
I don't think we need the `Result` here and I'd be fine with things failing silently (we don't currently have a way to see if the command was run successfully, even if we did have permissions).

I'd personally just return nothing here, but if you disagree, just let me know!

Reply:
I'd personally prefer to return something if we wait for the command exit, users may know if it's successful or not.
But right now, we don't wait for the command to finish, so I agree with your opinion for simplicity.

Reply:
@tw4452852 I totally agree :) I'd like to monitor the running of the command eventually, but I'm not sure how we'll do that just yet, so maybe it's best for a second PR :)

Reply:
I originally wanted to return the exit code which indicates the success or not. But I'm not sure if this is enough or general. Let's hold on to this for this moment now.